src/static/riot/competitions/form/_phases.tag

```diff
@@ -257,10 +257,15 @@
 
         self.one("mount", function () {
             // awesome markdown editor
-            self.simple_markdown_editor = new SimpleMDE({
+            self.simple_markdown_editor = new EasyMDE({
                 element: self.refs.description,
                 autoRefresh: true,
-                forceSync: true
+                forceSync: true,
+                renderingConfig: {
+                    markedOptions: {
+                        sanitize: true,
```

Comment:
Doesn't js hate hanging commas?

Reply:
https://stackoverflow.com/questions/7246618/are-trailing-commas-in-arrays-and-objects-part-of-the-spec

Looks like it's fine for JS, but not JSON?
Want me to pull it?